### DISCUSSION TITLE
Probing points 9 with new firmware 1.3.0

### DISCUSSION THREAD

**Discussion by ulivero** [Jyers/Marlin#468] [Q&A] [answered]
Hi i have Ender 3 v2 with 4.2.2 with bltouch , on last firmware the bltouch probes 25 points , with this new firmware only probes 9 .
Is there any way that the bltouch probes 25 points ?
Thanks in advance

**Comment by Jyers** [comment, marked as answer]
Are you compiling your own or using a precompiled, if so what version are you using? if you want 25 points you would need a 5x5 version.

**Reply by ulivero** [reply]
I compile my own , with templates that you post , theres only 3x3 bltouch.

**Reply by Jyers** [reply]
In configuration.h, change `#define GRID_MAX_POINTS_X 3` to `#define GRID_MAX_POINTS_X 5`

**Reply by ulivero** [reply]
Ok thanks as always Jyers

**Reply by ulivero** [reply]
its commented , i have tu uncommented?

**Reply by ulivero** [reply]
done

**Reply by Jyers** [reply]
You need to change the uncommented one, there are multiple for the different leveling modes

**Comment by smooreace** [comment]
More time reading before clicking me thinks
